aws_lambda_powertools/logging/types.py

```diff
@@ -0,0 +1,10 @@
+from typing import Any, Dict, Iterable, Union
+
```

Comment:
```suggestion
from __future__ import annotations
```

This will gives us a nicer syntax for Unions among other things. For example:

* `timestamp: Union[str, int]` becomes `timestamp: str | int`

If you'd like to learn more about this topic, check this out on how this enables Postponed Evaluation of Annotations (or type hint)

https://peps.python.org/pep-0563/

:)

Reply:
Out of curiosity, is there a reason why this isn't used for some of the other typing hints, like for return values on functions?